components/EntriesContainer/index.js

```diff
@@ -0,0 +1,29 @@
+import { styled } from "styled-components";
+import Entry from "../Entry";
+import { dummyData } from "@/pages/api/dummyData";
+
+function EntriesContainer() {
+  return (
+    <StyledEntrySection>
+      {dummyData.map((entry) => {
+        return (
+          <Entry
+            key={entry.slug}
+            JPDefinition={entry.japanese[0].word}
+            JPReading={entry.japanese[0].reading}
+            ENDefinition={entry.english_definitions}
```

Comment:
I know you'll end up with long prop names but...`japaneseDefinition`, `japaneseReading`, `englishDefinition`!

What's the difference between `word` and `definition` in the app? Why does it change from `japanese[0].word` to `japaneseDefinition`?

Also, call it `englishDefinitions` (plural), since you are passing an array ^^

Reply:
This is also from the API. I prefer "definition" for both. But the API uses "word" and no plural for the English definitions.